gateway-ha/src/main/java/io/trino/gateway/ha/config/SelfSignKeyPairConfiguration.java

```diff
@@ -1,8 +1,12 @@
 package io.trino.gateway.ha.config;
 
+import lombok.AllArgsConstructor;
```

Comment:
We are in the process of removing Lombok, ptal at @mosabua 's recent PRs. This will need to be refactored.

Reply:
I can go ahead and get rid of lombok here but since @mosabua already has a PR for it, I think it is best to keep this PR as is and merge this PR in before his PR to avoid merge conflicts